packages/buidler-core/src/builtin-tasks/node.ts

```diff
@@ -115,11 +115,7 @@ export default function () {
           console.log();
 
           try {
-            await watchCompilerOutput(
-              server.getProvider(),
-              getSolcVersion(config.solidity),
-              config.paths
-            );
+            await watchCompilerOutput(server.getProvider(), config.paths);
```

Comment:
The solcVersion was removed because now it's taken from the buildInfo file. Similar changes are elsewhere.